controllers/external/util.go

```diff
@@ -46,8 +49,10 @@ func Get(ctx context.Context, c client.Client, ref *corev1.ObjectReference, name
 	return obj, nil
 }
 
+type TemplateCloner struct{}
```

Comment:
We need to add some documentation in the v1a2->v1a3 docs about this change, given that it might be used elsewhere